### ISSUE TITLE
Some Questions about re-implement VectorNet

### ISSUE THREAD

**Issue by Hugh-Cai** [Henry1iu/TNT-Trajectory-Prediction#40]
Hi,

Are there some details (or tricks) when construct the model `vectornet`? I tried it according to the paper description, but the loss converges very slowly whatever learning rate. The loss term using `mse_loss` on `(x_offset, y_offset)` is 
```
Epoch: 0 | loss: 0.2395   ADE: 33.5584  FDE: 98.2625
Epoch: 1 | loss: 0.2312   ADE: 34.1547  FDE: 99.9695
Epoch: 2 | loss: 0.2270   ADE: 32.7290  FDE: 96.1623
Epoch: 3 | loss: 0.2246   ADE: 31.6781  FDE: 93.0649
Epoch: 4 | loss: 0.2222   ADE: 30.9941  FDE: 91.0348
...
```
(decay only very little per epoch,  and hyper-params is the same as the reference)

I‘m appreciate if you give me an answer!
Thanks

**Comment by Hugh-Cai** [comment]
Hi,

I used a standard MLP as decoder to predict the offsets (both x and y) in future 3 seconds, and used `mse_loss` as prediction loss and `smooth_l1_loss` as graph completion loss.

I implemented this only using `torch` without `torch_geometric`，so if did any questions appear in the encoder? Can I email the details to you if it's convenient？

Thanks for answering me!

Best Regards

**Comment by Henry1iu** [comment]
Hi,

The result (ADE and FDE) doesn't make sense if your model predicts the offsets.  I suggest you double-check your offset ground truth and the post-prediction processing (the offset accumulation). 

You can contact me via my email: henryliu@link.cuhk.edu.hk. I'll try my best to help you identify the problem : )

Best Regards,
Jianbang

**Comment by Hugh-Cai** [comment]
Thanks again!

And I described my understanding and details of implementation in the email. Please correct if I have any bias.

Best Regards

**Comment by JingyiShen99** [comment]
Hi, I have the same problem now, and I also implemented this only using torch without torch_geometric. Have you solved this problem? Could you give me some suggestions or ideas? 

Best Regards.
